hw/ip/lc_ctrl/data/lc_ctrl_sec_cm_testplan.hjson

```diff
@@ -144,19 +144,6 @@
       milestone: V2S
       tests: ["lc_ctrl_security_escalation"]
     }
-    {
-      name: sec_cm_main_ctrl_flow_consistency
-      desc: '''
-      MAIN.CTRL_FLOW.CONSISTENCY:
-
-      Verify that the control flow of the main FSM only ever progresses in one
-      direction - i.e. that there are no transitions that move backwards into
-      a state where the FSM has previously been (e.g. from `InvalidSt` back
-      into `IdleSt`).
-      '''
-      milestone: V2S
-      tests: []
```

Comment:
At the moment, this testpoints needs to be present since the tool does a crosscheck between the Hjson countermeasures list and the testplan (hence the CI failure).

@cindychip @weicaiyang how should we deal with countermeasures that are tested with assertions? Any ideas?